ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Test the connectivity for VIP's
+  command: ping -c3 -i 3 {{ apivip }} && ping -c3 -i 3 {{ ingressvip }} 
```

Comment:
you should make 2 different tasks for this and check for them separately. this could be a headache assuming both work or both fail